src/main/java/com/github/onsdigital/babbage/api/util/LegacyCacheProxyMaxAge.java

```diff
@@ -0,0 +1,64 @@
+package com.github.onsdigital.babbage.api.util;
+
+import java.io.IOException;
+
+import org.apache.http.Header;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import com.github.onsdigital.logging.v2.event.SimpleEvent;
+
+public class LegacyCacheProxyMaxAge {
+
+    private final String legacyCacheProxyUrl;
+
+    public LegacyCacheProxyMaxAge(String legacyCacheProxyUrl) {
+        this.legacyCacheProxyUrl = legacyCacheProxyUrl;
+    }
+
+    public int getMaxAgeValueFromLegacyCacheProxy(String resourceURI) {
+        RequestBuilder requestBuilder = buildRequest(resourceURI);          
```

Comment:
I think it would look a bit tidier if `buildRequest` called `build()` internally, and it returned the `HttpUriRequest` object directly. In fact, it may only need to return just `HttpRequest` instead of `HttpUriRequest`.